ksql-engine/src/test/java/io/confluent/ksql/planner/plan/JoinNodeTest.java

```diff
@@ -121,6 +122,10 @@
   @Rule
   public final ExpectedException expectedException = ExpectedException.none();
 
+  @Mock
```

Comment:
do we need a test for serde options of internal join topics?

Reply:
there are... see `shouldBuildLeftRowSerde` and `shouldBuildRightRowSerde`.